packages/wasm/src/services/credential/service.ts

```diff
@@ -55,6 +56,150 @@ export function isAnnonymousCredential(credential) {
   return isBBSPlusCredential(credential) || isKvacCredential(credential);
 }
 
+export async function createPresentationSDK(
+  vcs,
+  { challenge, domain, revealAttribs = [], witnesses = [], pexForBounds, presentationId, presentationOptions = {}, holderKey } = {}
+) {
+  const vcsArray = Array.isArray(vcs) ? vcs : [vcs];
+
+  const anoncredsVCs = vcsArray.filter(isAnoncredsProofType);
+  const nonAnoncredsVCs = vcsArray.filter((vc) => !isAnoncredsProofType(vc));
+  const revealAttribsArray = Array.isArray(revealAttribs[0]) ? revealAttribs : [revealAttribs];
+
+  const presentation = new VerifiablePresentation(presentationId);
+
+  let pexRequiredAttributes = [];
+  if (pexForBounds?.request) {
+    pexRequiredAttributes = getPexRequiredAttributes(
+      pexForBounds.request,
+      vcsArray,
+    );
+  }
+
+  await Promise.all(
+    anoncredsVCs.map(async (vc) => {
```

Comment:
this will create all anoncreds derived credentials at once, which could work, but in a large presentation use case it might result in node timeout errors. might be better to do them sequentially